packages/editor-ui/src/components/WorkflowShareModal.ee.vue

```diff
@@ -155,6 +155,7 @@ import { useWorkflowsEEStore } from '@/stores/workflows.ee';
 import { ITelemetryTrackProperties } from 'n8n-workflow';
 import { useUsageStore } from '@/stores/usage';
 import { BaseTextKey } from '@/plugins/i18n';
+import { isNavigationFailure } from 'vue-router';
```

Comment:
For learning, we cannot [set it up like in the docs](https://router.vuejs.org/api/#isnavigationfailure)?

Reply:
From what I've been able to test out, this hook is not triggered if navigation is canceled from another guard (such as `beforeRouteLeave`).